### PR TITLE
Splitting Can we do it on premise in two

### PR DESCRIPTION
**Pull request by darienmt**

### CODE REVIEW

File: content/chapter-01/CanWeDoItOnPremise.md [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-  * [Amazon Web Services](https://aws.amazon.com/products/): Compute, Storage, Database, Migration, Networking & Content Delivery, Developer Tools, Management Tools, Media Services, Security, Identity & Compliance, Analytics, Machine Learning, Mobile Services, AR & VR, ...
-  * [Microsoft Azure](https://azure.microsoft.com/en-ca/services/): Virtual Machines, App Service, SQL Database, Storage, Functions, Azure Cosmos DB, Azure Active Directory, Backup, HBInsight, Batch, Container Registry, Container Instances, Cloud Services, Azure Monitor, ...
-  * [Googlec Cloud](https://cloud.google.com/): Compute, Storage and Databases, Networking, Big Data, Data Transfer, API Platform & Ecosystems, Internet of Things, Machine Learning, Management Tools, Developer Tools, Identity & Security, ... 
-  
 
-* Bottom line:
-  * **Virtual Machines** are not the only aspect of a the Public Cloud (Ex. networks, recovery, applications, etc.)
-  * We can create a complete data center inside the Public Cloud, but there is more than or we may need much less
\ No newline at end of file
+* **Virtual Machines** are not the only aspect of a the Public Cloud (Ex. networks, recovery, applications, etc.)
```
Comment (vkhazin) [inline content/chapter-01/CanWeDoItOnPremise.md]: **Virtual Machines** is not the only aspect of a Public Cloud (e.g. networks, recovery, load-balancers, firewalls, etc.)

File: content/chapter-01/CanWeDoItOnPremise.md [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
+
+
+* We can create a complete data center inside the Public Cloud, but there is more than or we may need much less
```
Comment (vkhazin) [inline content/chapter-01/CanWeDoItOnPremise.md]: We can create a complete data centre on a Public Cloud, but there is more than that or, we may need much less than that

File: content/chapter-01/PublicCloudProvidersOffers.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Why I cannot do it all in my own data centre?
+
+## Short product list offered by Public Cloud providers
+
+* [Amazon Web Services](https://aws.amazon.com/products/): Compute, Storage, Database, Migration, Networking & Content Delivery, Developer Tools, Management Tools, Media Services, Security, Identity & Compliance, Analytics, Machine Learning, Mobile Services, AR & VR, ...
```
Comment (vkhazin) [inline content/chapter-01/PublicCloudProvidersOffers.md]: * [Amazon Web Services](https://aws.amazon.com/products/): Compute, Storage, Database, Migration, Networking & Content Delivery, Developer Tools, Management Tools, Media Services, Security, Identity & Compliance, Analytics, Machine Learning, Mobile Services, AR & VR, and counting...

File: content/chapter-01/PublicCloudProvidersOffers.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+* [Microsoft Azure](https://azure.microsoft.com/en-ca/services/): Virtual Machines, App Service, SQL Database, Storage, Functions, Azure Cosmos DB, Azure Active Directory, Backup, HBInsight, Batch, Container Registry, Container Instances, Cloud Services, Azure Monitor, ...
```
Comment (vkhazin) [inline content/chapter-01/PublicCloudProvidersOffers.md]: [Microsoft Azure](https://azure.microsoft.com/en-ca/services/): Virtual Machines, App Services, SQL Database, Storage, Functions, Azure Cosmos DB, Azure Active Directory, Backup, HBInsight, Batch, Container Registry, Container Instances, Cloud Services, Azure Monitor, and the list goes on...

File: content/chapter-01/PublicCloudProvidersOffers.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+* [Googlec Cloud](https://cloud.google.com/): Compute, Storage and Databases, Networking, Big Data, Data Transfer, API Platform & Ecosystems, Internet of Things, Machine Learning, Management Tools, Developer Tools, Identity & Security, ...
```
Comment (vkhazin) [inline content/chapter-01/PublicCloudProvidersOffers.md]: [Google Cloud](https://cloud.google.com/): Compute, Storage and Databases, Networking, Big Data, Data Transfer, API Platform & Ecosystems, Internet of Things, Machine Learning, Management Tools, Developer Tools, Identity & Security, and it is expanding every so often...

File: content/chapter-01/CanWeDoItOnPremise.md (hunk 1 above)
Comment (darienmt) [reply content/chapter-01/CanWeDoItOnPremise.md]: Done

File: content/chapter-01/CanWeDoItOnPremise.md (hunk 2 above)
Comment (darienmt) [reply content/chapter-01/CanWeDoItOnPremise.md]: Done

File: content/chapter-01/PublicCloudProvidersOffers.md (hunk 3 above)
Comment (darienmt) [reply content/chapter-01/PublicCloudProvidersOffers.md]: Done

File: content/chapter-01/PublicCloudProvidersOffers.md (hunk 4 above)
Comment (darienmt) [reply content/chapter-01/PublicCloudProvidersOffers.md]: Done

File: content/chapter-01/PublicCloudProvidersOffers.md (hunk 5 above)
Comment (darienmt) [reply content/chapter-01/PublicCloudProvidersOffers.md]: Done
